test.sh

```diff
@@ -18,7 +18,7 @@ for x in Dockerfile*;do
   echo "======================="
   docker build -t runtimetestimage -f "${x}" .
   #passes an env variable of size more than 4K chars
-  time dgoss run -v `pwd`/test_scripts:/test_scripts -e TEST_LONG_VAR="${largevar}" runtimetestimage || ((i++))
+  time dgoss run -v `pwd`/test_scripts:/test_scripts -e TEST_LONG_VAR="${largevar}" -e 1VAR_NAME=varnameStartWithNumber -e var.name=varnameHasPeriod -e var-name=varnameHasHyphen -e _=varnameHasJustUnderscore runtimetestimage || ((i++))
```

Comment:
According to: https://www.gnu.org/software/bash/manual/html_node/Special-Parameters.html

```
($_, an underscore.) At shell startup, set to the absolute pathname used to invoke the shell or shell script being executed as passed in the environment or argument list. Subsequently, expands to the last argument to the previous command, after expansion. Also set to the full pathname used to invoke each command executed and placed in the environment exported to that command. When checking mail, this parameter holds the name of the mail file.
```

So I guess normally, `_` in this particular case would be ... `/path/to/docker` ?